modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -0,0 +1,476 @@
+
+Redpanda supports https://iceberg.apache.org/spec/#format-versioning[version 2^] of the Iceberg table format.
+
+== Iceberg concepts 
+
+https://iceberg.apache.org[Apache Iceberg^] is an open source format specification for defining structured tables in a data lake. The table format lets you quickly and easily manage, query, and process huge amounts of structured and unstructured data. This is similar to the way in which you would manage and run SQL queries against relational data in a database or data warehouse. The open format lets you use many different languages, tools, and applications to process the same data in a consistent way, so you can avoid vendor lock-in. This data management system is also known as a _data lakehouse_.
+
+In the Iceberg specification, tables consist of the following layers:
+
+* Data layer: Stores the data in data files. The Iceberg integration currently supports the Parquet file format. Parquet files are column-based and suitable for analytical workloads at scale. They come with compression capabilities that optimize files for object storage. 
+* Metadata layer: Stores table metadata separately from data files. The metadata layer allows multiple writers to stage metadata changes and apply updates atomically. It also supports database snapshots, and time travel queries that query the database at a previous point in time. 
++
+--
+** Manifest files: Track data files and contain metadata about these files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+For Iceberg-enabled topics, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. The Iceberg integration supports two xref:manage:iceberg/use-iceberg-catalogs.adoc[catalog integration] types. You can configure Redpanda to catalog files stored in the same object storage bucket or container where the Iceberg data files are located, or you can configure Redpanda to use an https://iceberg.apache.org/terms/#decoupling-using-the-rest-catalog[Iceberg REST catalog^] endpoint to update an externally-managed catalog when there are changes to the Iceberg data and metadata.
+
+// image::shared:iceberg-integration-optimized.png[Redpanda's Iceberg integration]
+image::ROOT:shared:iceberg-integration-optimized.png[Redpanda's Iceberg integration]
```

Comment:
The image doesn't display in the preview

Reply:
Updated to add the same image in cloud-docs